lightning/src/ln/functional_tests.rs

```diff
@@ -7282,8 +7282,20 @@ fn test_user_configurable_csv_delay() {
 	} else { assert!(false); }
 }
 
-#[test]
-fn test_data_loss_protect() {
+/// describe the test case as a enum, istead as boolean in this case!
```

Comment:
I'm too verbose with the comments because I tend to forget what I did. if are too much I will remove it